baseapp/baseapp_test.go

```diff
@@ -192,6 +193,12 @@ func setupBaseAppWithSnapshots(t *testing.T, blocks uint, blockTxs int, options
 	return app, teardown
 }
 
+func shouldPanic(t *testing.T, msg string, f func()) {
+	defer func() { recover() }()
+	f()
+	t.Errorf(msg)
+}
+
```

Comment:
For future reference, there's a `.Panics` in assert/require. Also `.PanicsWithError` and `.NotPanics`.

Reply:
good points